bundles/core-combat/player-events.js

```diff
@@ -17,12 +17,12 @@ module.exports = (srcPath) => {
         // could potentially wind up in a situation where the player performed
         // a mid-round attack that killed the target, then the next round the
         // target kills the player. So let's not let that happen.
-        if (this.attributes.health <= 0) {
+        if (this.getAttribute('health') <= 0) {
           this.combatants.forEach(combatant => {
             this.removeCombatant(combatant);
             combatant.removeCombatant(this);
             if (!combatant.isInCombat()) {
-              combatant.attributes.health = combatant.attributes.maxHealth;
+              combatant.setAttribute('health', combatant.getRawAttribute('maxHealth'));
```

Comment:
I may have mistyped, it would set their actual `health` to more than their actual `maxHealth`

Effect: cast a spell to double `maxHealth`
actual `health` && effective `health`: 100
actual `maxHealth`: 100
effective `maxHealth`: 200

They appear to be at 50% health now.

They use a healing potion to boost their health up to the new effective maximum, assuming use of `getAttribute('maxHealth')`:

actual `health`: 200
effective `health`: 200
actual `maxHealth`: 100
effective `maxHealth`: 200

They now appear to have 100% health.

The effect wears off. If the effect's deactivation does not account for `health` and merely reverts `maxHealth`, then we end up with this scenario:

actual `health`: 200
effective `health`: 200
actual `maxHealth`: 100
effective `maxHealth`: 100

They now appear to have 200% health and become demi-gods.